src/main/java/com/cburch/logisim/std/ttl/Ttl74244.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Logisim-evolution - digital logic design tool and simulator
+ * Copyright by the Logisim-evolution developers
+ *
+ * https://github.com/logisim-evolution/
+ *
+ * This is free software released under GNU GPLv3 license
+ */
+
+package com.cburch.logisim.std.ttl;
+
+/**
+ * TTL 74x244 octal buffers and line drivers with three-state outputs
```

Comment:
Why don't you use the more detailed description you added to `std.properties`?

Reply:
this is basically the plain vanilla version, so the description is the same